actix/src/sec-5-arbiter.md

```diff
@@ -15,89 +15,148 @@ for your actors to run inside. When you call `start()` on your actor it is then
 running inside of the System Arbiter's thread. In many cases, this is all you
 will need for a program using Actix.
 
+While it only uses one thread, it uses the very efficient event loop pattern
+which works well for asynchronous events. To handle synchronous tasks that
+require heavy CPU power, it's better to avoid blocking the event loop and
+instead offload the computation to other threads. For this usecase, read the
+next section and consider using [`SyncArbiter`](./sec-6-sync-arbiter.md).
+
+## The event loop
+
+> If you're familiar with event loops in other paradigms, such as libuv (upon
+> which Node runs) or tokio (upon which Actix is implemented), each Arbiter has
+> a similar processing paradigm.
+
+> Specifically, if you're familiar with how JavaScript handles asynchronicity
+> and how the single-threaded model makes it so JS mostly doesn't need
+> synchronization primitives, this should feel very familiar to you.
+
+One `Arbiter` is in control of one thread with one task queue, and when tasks
+get spawned in an Actor's context or using `Arbiter::spawn`, the task gets
+queued up for execution in that thread.
+
+If you pay close attention to way both `ActorFuture` and the methods in the
+`Actor` trait specify their parameters, actor behavior always references both
+the actor and the context as mutable: `&mut self` (or `&mut A` in `ActorFuture`)
+and `&mut Self::Context`. This is because all of these tasks have **exclusive**
+rights to the actor and its context. This is achieved by putting all actor logic
+in the same thread, so we know no two pieces of logic involving the actor are
+running concurrently without having to use synchronization primitives.
+
+While the library is written with an asynchronous, event-driven flow, and while
```

Comment:
I think this paragraph is just repeating all the content above it, so it may not be needed.

Reply:
I agree but I think I'll reorient it to emphasize that Actix *can* work in an asynchronous fashion, but you have to manually handle Arbiters yourself.